### DISCUSSION TITLE
Flutter Release build on real device

### DISCUSSION THREAD

**Discussion by ifeanyidotdev** [rikulo/socket.io-client-dart#389] [Q&A]
i am working with this client on a flutter for mobile, if i build release build and run the release build on an emulator the websocket works fine, but if i run the release build on a real device , the logs shows that the connection was successful but the websocket does not do the intended for realtime data on the real device

**Comment by yajatkaul** [comment]
it is either that the connection is unsecure or something try to log the data to confirm if you even receive it
